### PR TITLE
Contribution: Education - first part of Translation GERMAN of: "Chapter XIII Developers Intro Tutorial"

### PR DESCRIPTION
**Pull request by mARTin-B78**

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Tutorials/A/Aggregate/Aggregate-data-from-different-Exchanges-in-a-Chart-/aggregate-data-from-different-exchanges-in-a-chart-001-combine-exchanges-in-a-chart-tutorial.json [hunk 1]
```diff
@@ -7,22 +7,22 @@
         "updated": 1617376996248,
         "translations": [
             {
-                "language": "ES",
-                "text": "Este Tutorial cubre el proceso de reunir el conjunto de datos más diverso: datos de distintos exchanges! Al final de este tutorial, podrá crear gráficos casi imposibles de encontrar en ningún otro lugar..",
-                "updated": 1636367640855
+                "language": "DE",
+                "text": "In diesem Tutorial lernen Sie, wie Sie die unterschiedlichsten Daten zusammenstellen können: Daten von verschiedenen Börsenplätzen! Am Ende dieses Tutorials werden Sie in der Lage sein, Diagramme zu erstellen, die sonst nirgendwo zu finden sind.",
+                "updated": 1641764147986
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Tutorials/A/Aggregate/Aggregate-data-from-different-Exchanges-in-a-Chart-/aggregate-data-from-different-exchanges-in-a-chart-001-combine-exchanges-in-a-chart-tutorial.json:12]: Hey @mARTin-B78 du überschreibst ausversehen die Spanische Übersetzung, kannst du das "reparieren"? :)

**Comment by harrellbm** [comment]
@mARTin-B78 There are still a lot of overwrite errors I'm guessing someone has just been working on this as well as you

**Comment by gitforkgit** [comment]
@mARTin-B78 weißt du wie man die probleme behebt?

**Comment by mARTin-B78** [comment]
have resolved the issues.

hope that works now

**Review by mARTin-B78 — COMMENTED** [review]
no clue why that stuff happens but it should not matter

**Review by mARTin-B78 — COMMENTED** [review]
no clue why that happens but it should not matter

**Comment by harrellbm** [comment]
@mARTin-B78 @gitforkgit The only file that looks like it's still showing up as having an issue is Projects/Foundations/Schemas/Docs-Tutorials/A/Aggregate/Aggregate-data-from-different-Exchanges-in-a-Chart-/aggregate-data-from-different-exchanges-in-a-chart-001-combine-exchanges-in-a-chart-tutorial.json.

If you are having trouble fetching and merging that means there must be a merge conflict that is not showing up on GitHub for some reason.  The best way to try and solve it would be to use vs code to try and merge in the main repo manually so you can see all the errors and what's going wrong

**Comment by gitforkgit** [comment]
Projects/Foundations/Schemas/Docs-Tutorials/A/Aggregate/Aggregate-data-from-different-Exchanges-in-a-Chart-/aggregate-data-from-different-exchanges-in-a-chart-001-combine-exchanges-in-a-chart-tutorial.json
@mARTin-B78  This file still shows up wrong for me.

**Review by mARTin-B78 — COMMENTED** [review]
There's been a } Bracket replaced by a } Bracket - not sure how and why this happens but it should not matter - or does it?

**Comment by gitforkgit** [comment]
@mARTin-B78 The replacement of the spanish version still shows up to me. I have no Idea how this happens or how to resolve this conflict. @harrellbm

**Comment by gitforkgit** [comment]
As it seems like only that one file makes problems, you can back it up and delete it from this Pull request so we can merge the rest? @harrellbm what do you think?

**Comment by harrellbm** [comment]
@mARTin-B78 @gitforkgit I think like you said backing up and restoring the original is the way to go.  Then after this is merged you should be able to add back the translations for that file

**Comment by gitforkgit** [comment]
@mARTin-B78 can you backup this file and delete, or is there something messed up that this doesnt work?
Let me know if errors occur.

**Comment by gitforkgit** [comment]
@mARTin-B78 could you maybe delete the one conflicting file from the pull request and resolve any conflict that may have occured inbetween here on github as this could make it possible to merge this pull request. :+1:

**Pull request closed without merging**